vault/resource_namespace_test.go

```diff
@@ -45,6 +45,15 @@ func TestAccNamespace(t *testing.T) {
 		Providers:    testProviders,
 		CheckDestroy: testNamespaceDestroy(namespacePath),
 		Steps: []resource.TestStep{
+			{
+				Config: testNamespaceConfig(namespacePath),
+				Check:  testNamespaceCheckAttrs(),
+			},
+			{
+				Config: testNamespaceConfig(namespacePath + "-foo"),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("vault_namespace.test", consts.FieldPath, namespacePath+"-foo")),
```

Comment:
I wonder if we should also check to that the old namespace was properly deleted in Vault? You can do that in another `TestStep`? I think `testNamespaceDestroy()` might work here.

Reply:
Great idea, I was actually thinking about adding that, and so will do so!

Reply:
I added this using `testNamespaceDestroy()` as an additional `Check` as part of the second step and it looks like it works to check that the original namespace with that path was destroyed. I wasnt sure what to use for the `Config` portion of an additional `TestStep` yet, but if you have ideas I'd love to chat offline!